src/TextField/TextFieldLabel.spec.js

```diff
@@ -3,63 +3,35 @@
 
 import React from 'react';
 import { assert } from 'chai';
-import { createShallowWithContext } from 'test/utils';
+import { createMountWithContext } from 'test/utils';
 import TextFieldLabel, { styleSheet } from './TextFieldLabel';
 
 describe('<TextFieldLabel />', () => {
-  let shallow;
+  let mount;
   let classes;
 
   before(() => {
-    shallow = createShallowWithContext();
-    classes = shallow.context.styleManager.render(styleSheet);
-  });
-
-  it('should render a <label />', () => {
-    const wrapper = shallow(<TextFieldLabel className="woof" />);
-    assert.strictEqual(wrapper.is('label'), true, 'should be a <label>');
-    assert.strictEqual(wrapper.hasClass(classes.root), true, 'should have the root class');
-    assert.strictEqual(wrapper.hasClass('woof'), true, 'should have the user class');
+    mount = createMountWithContext();
```

Comment:
I'm totally unfamiliar and looking at enzyme now.  If we don't need mount that's good to know as I'm trying to figure that out now.